src/monodroid/jni/application_dso_stub.cc

```diff
@@ -117,3 +119,22 @@ AssemblyStoreRuntimeData assembly_stores[] = {
 		.assemblies = nullptr,
 	},
 };
+
+constexpr char fake_dso_name[] = "libaot-Some.Assembly.dll.so";
```

Comment:
Why have *any* `DSOCacheEntry` values in the "stub" build?  Why not just a zero-length array?  (Or does something require non-zero length arrays?)

Reply:
I'm using disassembly of the stub to make sure our managed generator does the right job, for that I need it to have "valid" content.